### ISSUE TITLE
FR: Suggest optimizing PHP version comparisons

### ISSUE THREAD

**Issue by brandonkelly** [kalessil/phpinspectionsea#517]
Would be cool if the plugin found `version_compare()` calls on the `PHP_VERSION` and suggested a simpler numeric comparison using `PHP_VERSION_ID` (and provided a quick fix to automatically fix them).

Example:

```php
version_compare(PHP_VERSION, '7.1', '>=')
```

could be optimized to

```php
PHP_VERSION_ID >= 70100
```

**Comment by kalessil** [comment]
Sure thing @brandonkelly : will be in the next release =)

**Comment by kalessil** [comment]
Implemented, 2.3.11 ETA is ~mid of October
